### ISSUE TITLE
Patch update message is incorrect

### ISSUE THREAD

**Issue by PieterBoeren** [single-spa/create-single-spa#285]
Minor issue, but when running "create-single-spa" command, I get this message:

```
"A patch update of create-single-spa is available: 2.3.1 → 2.2.2"
```

It seems that the upgrade check is only taking the patch version into consideration and not the entire version.

**Comment by joeldenning** [comment]
Thanks for the bug report, I have fixed it in #290

**Comment by joeldenning** [comment]
Fix released in create-single-spa@2.2.3. See https://github.com/single-spa/create-single-spa/blob/main/packages/create-single-spa/CHANGELOG.md
